src/main/java/de/uhd/ifi/se/decision/management/jira/model/KnowledgeElement.java

```diff
@@ -672,6 +673,14 @@ public String getStatusAsString() {
 		return getStatus().toString();
 	}
 
+	public int getLineCount() {
```

Comment:
Do you need the line count in the KnowledgeElement class or can it be moved to the ChangedFile class?

Reply:
Since `CodeCompletenessCheck` implements `CompletenessCheck`, which works on `KnowledgeElement`s in general (and so does its `execute()` function), I would need to convert the `KnowledgeElement` into a `ChangedFile`, which I tried using a `GitClient`, but this performed very badly. I know that this is not very pretty, but for the time being this seemed to be the best approach to me.

Reply:
In addition, the line count needs to be handled by the `CodeClassPersistenceManager`, which also uses `KnowledgeElement` instead of `ChangedFile` in many cases.